lib_sw_pll/src/sw_pll_common.c

```diff
@@ -77,7 +77,7 @@ void sw_pll_app_pll_init(const unsigned tileid,
 #define APP_PLL_FRAC_12M 0x80000104
 
 #define APP_PLL_CTL_11M  0x09009100
-#define APP_PLL_DIV_11M  0x80000009
```

Comment:
It'd be good to get the comments from the output of the pll_calc.py in here too as per other entires